packages/styles/src/tile.scss

```diff
@@ -831,25 +845,44 @@ $fd-line-tile-badge-horizontal-spacing: 0.375rem !default;
       }
 
       .#{$block}__action-close {
-        @include fd-set-height(1.625rem);
-        @include reset-position-absolute();
-        @include fd-flex-center();
-
+        @include fd-set-height(1.375rem);
+        @include fd-set-width(1.375rem);
+        position: relative;
         padding-block: 0;
         padding-inline: 0.25rem;
+        font-size: 0.75rem;
+        border-radius: 50%;
+        background: var(--sapButton_Background);
+        border: var(--sapButton_BorderColor);
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        cursor: pointer;
+        top: -0.625rem;
+        right: -0.325rem;
+
+        &::after {
+          border-radius: 50%;
+        }
       }
 
       .#{$block}__action-indicator {
         margin-inline: 0;
         margin-block: 0;
-      }
 
-      .#{$block}__action-indicator,
-      .#{$block}__action-close {
-        @include reset-position-absolute();
-
-        @include fd-focus() {
-          @include fd-button-focus(0);
+        position: relative;
+        height: 1.625rem;
+        width: 1.625rem;
```

Comment:
when width and height are the same you can use one of the mixins:
```
@mixin fd-square($size) {
  height: $size;
  width: $size;
  min-width: $size;
}

@mixin fd-set-square($size) {
  width: $size;
  min-width: $size;
  max-width: $size;
  height: $size;
  min-height: $size;
  max-height: $size;
}

```

Reply:
done!